content/docs/user-guide/programming/debugging/crash-dumps.md

```diff
@@ -1,5 +1,27 @@
 ---
-title: "Crash Dumps"
-date: 2021-03-02T00:23:53-05:00
+title: Crash Reporting with Dump Files
+linktitle: Crash Reporting
+description: When you encounter a crash in Open 3D Engine (O3DE), it's helpful to generate a dump file with crash reporting information to help track down the problem. 
 ---
 
+If you encounter a crash with **Open 3D Engine (O3DE)**, you may find helpful information in the dump file, `error.dmp`. The dump file records details about the state of the engine that led to the event of the crash. The dump file is a compact minidump and is located in the `user/log/` directory, within the same parent directory where you ran the O3DE Editor from. For example, if you ran the Editor from my project's directory, then you can find the dump files in `/MyProject/user/log/`. 
```

Comment:
For example, if you ran the Editor from the MyProject's directory, then you can find the dump files in `/MyProject/user/log/`.

Reply:
Created a new PR to resolve this.